airbyte_cdk/sources/declarative/interpolation/macros.py

```diff
@@ -177,6 +177,7 @@ def format_datetime(
         dt_datetime = (
             datetime.datetime.strptime(dt, input_format) if input_format else str_to_datetime(dt)
         )
+    dt_datetime = dt_datetime.replace(tzinfo=pytz.utc)
```

Comment:
Should we document that we are now forcing UTC?

Reply:
From the thread, should we have a replace method in DatetimeParser that leverages _is_naive? So we can reuse.

Reply:
@maxi297 - Can we make this conditional? Otherwise a valid timezone coming from the source API may be replaced.

Something like this?:

```suggestion
    if dt_datetime.tzinfo is None:
        dt_datetime = dt_datetime.replace(tzinfo=pytz.utc)
```

Reply:
You could alternatively wrap the `ab_datetime_format ` datetime helper. It already has this behavior.

https://airbytehq.github.io/airbyte-python-cdk/airbyte_cdk/utils/datetime_helpers.html#ab_datetime_format

Reply:
Yes, that makes sense. I intended to do that yesterday but travelling got in the way. I'll add a test for that too and ping you again for a review